lib/LaTeXML/Package/icml_support.sty.ltxml

```diff
@@ -19,8 +19,13 @@ use LaTeXML::Package;
 RequirePackage('times');
 RequirePackage('fancyhdr');
 RequirePackage('color');
-RequirePackage('algorithm');
-RequirePackage('algorithmic');
+DefPrimitive('\icml@fallback@algorithm', sub {
+    # Load algorithmic, but only if not a version of it has been explicitly loaded already.
+    if (!IsDefined(T_CS('\algorithmic'))) {
+      RequirePackage('algorithm');
+      RequirePackage('algorithmic'); }
+    return; });
+PushValue('@at@begin@document', T_CS('\icml@fallback@algorithm'));
```

Comment:
This part doesn't seem necessary, since icml itself, simply loads the two packages. If there's a conflict, it should probably be handled by the algorithmic(x) packages.